services/web-app/components/anchor-links/anchor-links.js

```diff
@@ -21,7 +21,7 @@ import styles from './anchor-links.module.scss'
  * content is long enough.
  */
 const AnchorLinks = ({ children, small, className }) => {
-  const isColumn = Children.count(children) > 6
+  const isColumn = Children.count(children) > 9
```

Comment:
after how many links is this supp to break into a new column? it's breaking at 4 for me, correct?
<img width="692" alt="image" src="https://user-images.githubusercontent.com/40550942/167061568-b47fd3da-4e3e-465f-8a6d-5a2ec3fd38a7.png">

Reply:
This is correct